game/src/ui/gameplay/towerInventory/TowerInventory.tscn

```diff
@@ -1,10 +1,11 @@
-[gd_scene load_steps=9 format=2]
+[gd_scene load_steps=10 format=2]
 
```

Comment:
I found a bug regarding tower inventory: 

https://user-images.githubusercontent.com/7360964/123512952-e048df80-d6bc-11eb-903b-4afb99c4699a.mp4

Reply:
Don't fix this just yet tho. We should discuss how we're fixing the scaling issue

Reply:
fixed aspect ratio to be constant